### DISCUSSION TITLE
Pregunta Practicas

### DISCUSSION THREAD

**Discussion by AlejoAlzate3** [Launch-X-Latam/MisionFrontEnd#13] [Q&A]
¿Las practicas las debemos subir algún repositorio o solo se realizan y se tienen guardadas?

**Comment by JosafatJimenezB** [comment]
Las practicas las vas a subir a un repositorio, estas las pedirán al final para poder pasar a la sección de backend además de que te sirven para demostrar todo lo que aprendiste aquí, saludos!

**Reply by AlejoAlzate3** [reply]
Josafat, muchas gracias, saludos.

**Reply by JosafatJimenezB** [reply]
denada amigo
